patches/chrome-browser-component_updater-widevine_cdm_component_installer.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/component_updater/widevine_cdm_component_installer.cc b/chrome/browser/component_updater/widevine_cdm_component_installer.cc
+index 650e041feaaa0e83d71b4207fd10d5390845f476..c79bdd8ad18de00057594964b2447a251f736a15 100644
+--- a/chrome/browser/component_updater/widevine_cdm_component_installer.cc
++++ b/chrome/browser/component_updater/widevine_cdm_component_installer.cc
+@@ -75,6 +75,7 @@ const char kWidevineCdmPlatform[] =
+ const char kWidevineCdmArch[] =
+ #if defined(ARCH_CPU_X86)
+     "x86";
++#if defined(ARCH_CPU_ARM64) && defined(OS_APPLE) "x64";
```

Comment:
this will go away when an arm64 widevine lib is available from the google updater